packages/aquedux-server/src/network/routes/fragments.js

```diff
@@ -1,14 +1,17 @@
+import configManager from '../../managers/configManager'
 import { selectors } from '../../reducers'
 
+const { queueLimit } = configManager.getConfig()
+
 export const getFragmentsInfo = store => {
   const queueNames = Object.keys(selectors.queue.listQueues(store.getState()))
+  const cursor = selectors.queue.getCursor(store.getState(), channelName)
+  const index = queueLimit === 0 ? cursor : Math.floor(cursor / queueLimit)
```

Comment:
Same as before, those lines could be merged with packages/aquedux-server/src/network/routes/fragment.js:11 logic.